### ISSUE TITLE
ARM64 Support

### ISSUE THREAD

**Issue by drxjason** [AdrianWilczynski/OneDarkPro#33]
Hi, i am running visual studio 2022 on arm64. i would love to see an arm64 version of this theme. thanks

**Comment by Tiktack** [comment]
Yeah, it's not that hard to add additional target. Can we ger update for this?

**Comment by standamikes** [comment]
I would also really appreciate it. Thank you so much in advance  :)
